c_horiz_interp/c_horiz_interp.F90

```diff
@@ -4,6 +4,8 @@ module c_horiz_interp_mod
   use FMS, only : FmsHorizInterp_type
   use FMS, only : fms_horiz_interp_new
   use FMS, only : fms_string_utils_c2f_string
+  use FMS, only : fms_mpp_error, FATAL
+  use horiz_interp_type_mod, only : CONSERVE, BILINEAR
```

Comment:
where are these parameters used?
Can they be exportable so that they're available as cFMS parameters?
(see https://github.com/NOAA-GFDL/cFMS/tree/83b6b919887302f5783156a7aacfcca057106da0/c_constants)

Reply:
I was using them earlier but took that part out, I just removed the imports because they shouldn't be needed externally.